### PR TITLE
Add CVE-2019-5624

### PR DESCRIPTION
**Pull request by todb-r7**
Metasploit boog.

### CODE REVIEW

File: 2019/5xxx/CVE-2019-5624.json [hunk 1]
```diff
@@ -1,18 +1,119 @@
 {
```
Comment (cve-team) [inline 2019/5xxx/CVE-2019-5624.json]: A language tag was used multiple times: problemtype value #0

Comment (todb-r7) [reply 2019/5xxx/CVE-2019-5624.json]: Hmm, this error message has me a little confused. Should this:

```xml
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "eng",
            "value": "CWE-22"
          },
          {
            "lang": "eng",
            "value": "Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal')"
          }
        ]
      }
    ]
  },
```

Really be just this:

```xml
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "eng",
            "value": "CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal')"
          }
        ]
      }
    ]
  },
```

For some reason I had it in my head that the CWE-NNN and name text should be different elements, but maybe that's wrong, and that's what the linter here is really complaining about.

Let me know, @cve-team !

Comment (cve-team) [reply 2019/5xxx/CVE-2019-5624.json]: Hello Tod,

It should be the latter CWE-NNN and Name. We can suggest a change to the schema to add an additional field that can be used to separate the ID from the name. Maybe a desc tag?

Thanks,
Anthony S.
CVE Assignment Team

Comment (todb-r7) [reply 2019/5xxx/CVE-2019-5624.json]: Yeah, it feels weird to have two "things" (colon separated) smooshed into one element. But that's for another day, I'll update to the current standard.

Comment (todb-r7) [reply 2019/5xxx/CVE-2019-5624.json]: Fixed!

**Pull request merged**